AudioBar/AudioBarViewController.swift

```diff
@@ -96,6 +108,43 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
         }
     }
 
+    private func configureCommandCenterCommands() {
+        commandCenter.playCommand.addTarget { [weak store] _ in
```

Comment:
Well, sure. It's a play button after all. I'm open to anything that's clean, reads well, makes sense and removes logic from the imperative shell.